test-suite/js/src/test/scala/org/scalajs/testsuite/library/JsBigIntTest.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.JSAssert.assertJSUndefined
+
+import scala.scalajs.js
+
+class JsBigIntTest {
```

Comment:
This test class will need a companion object with an `@BeforeClass` def to assume that we're running with Node.js or in the browser, as neither Rhino nor PhantomJS support BigInts.

Reply:
(also it should just be called `BigIntTest`, not `JsBigIntTest`, since it contains the tests for the class `BigInt`, not `JsBigInt`.)